### PR TITLE
Add MV.py, CLEAR.py and fixes some stuff

### PR DESCRIPTION
**Pull request by OcelotWalrus**
# Fixes + Progression

## Summary

This PR fixes some bugs happening when some functions weren't run in the root directory. This PR also updates the pwd.py command output.

It also adds two new commands: mv.py (move files or dirs) and clear.py (clear the prompt).
Note that for now, the mv.py command doesn't work with `../` paths-like.

### CODE REVIEW

File: kernel/master.py
Comment (Monarchz3ro) [inline kernel/master.py:1]: I do not understand why this change was made. I am not saying you are wrong, I simply do not understand why.

File: kernel/root/bin/clear.py
Comment (Monarchz3ro) [inline kernel/root/bin/clear.py:1]: This only works on unix systems, right? I intend to make this work for Windows first. Is there a way to detect what OS the system is running?

File: kernel/master.py
Comment (Monarchz3ro) [reply kernel/master.py:1]: After putting thought into the PR and trying this on my own system, I understand why. Thanks for this.

**Comment by OcelotWalrus** [comment]
I've added a new commit that adds a executable check before running any command!

(check if the current user as a 'x' permission in the file dictionary in the ecosystem.json file)

**Comment by OcelotWalrus** [comment]
New commit that adds the `/home/` dir and the `root/` directory. It's more like a linux filesystem now. And like that, users have their own directory so they can actually do stuff.
In every home folder I've added a `.bashrc` file so there aren't empty.

Is that good?

File: kernel/master.py
Comment (OcelotWalrus) [reply kernel/master.py:1]: You're welcome! The problem that if you weren't on the root folder, it would crash.

File: kernel/root/bin/clear.py
Comment (OcelotWalrus) [reply kernel/root/bin/clear.py:1]: No, it should work for every system.

**Comment by OcelotWalrus** [comment]
Also @Monarchz3ro , I  advise you to merge using squash merge, it's cleaner. Squash merge create a single commit from all the PR commits and put it in the master branch.

**Comment by Monarchz3ro** [comment]
> Also @Monarchz3ro , I advise you to merge using squash merge, it's cleaner. Squash merge create a single commit from all the PR commits and put it in the master branch.

I see. Thank you.
